onyo/commands/get.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import sys
+
+from rich import box
+from rich.console import Console
+from rich.table import Table
+
+from onyo import Repo, OnyoInvalidRepoError
+
+logging.basicConfig()
+log = logging.getLogger('onyo')
+
+
+def get(args, opdir):
+    """
+    Display the requested `key`(s) in tabular form for matching assets.
+    """
+    try:
+        repo = Repo(opdir)
+        repo.fsck(['asset-yaml'])
+    except OnyoInvalidRepoError:
+        sys.exit(1)
+
+    # str->list arguments (comma-separated)
+    # note: "foo, bar, baz" == foo,bar,baz
+    keys = list(key.strip() for key in args.keys.split(','))
+    paths = set(path.strip() for path in args.path.split(','))
```

Comment:
Initially, consistency. But now that you've mentioned it, I realize that valid paths can contain commas in file or directory names which would mess this up. I've addressed this by changing the `path` argument to have `nargs='*'` (for 0 or more entries, since `default='.'`) and removing the comma separation.

Reply:
As discussed on video chat, all comma-separation has been removed for keys, paths, and filters. Instead `--options` are used for each of them.